src/Microsoft.Health.Operations.Functions/Management/PurgeOrchestrationInstanceHistory.cs

```diff
@@ -57,27 +55,39 @@ public async Task Run([TimerTrigger(PurgeFrequencyVariable)] TimerInfo myTimer,
         EnsureArg.IsNotNull(client, nameof(client));
         EnsureArg.IsNotNull(myTimer, nameof(myTimer));
         EnsureArg.IsNotNull(log, nameof(log));
+        IReadOnlyCollection<OrchestrationRuntimeStatus>? statuses = EnsureArg.HasItems(_options.Statuses, nameof(_options.Statuses));
```

Comment:
Can't we assert these in the ctor? I don't think we need to assert these everytime, although if someone keeps the reference I suppose it could wreck havoc.

As an aside, we're mostly making these checks for atypical cases, as the data annotations should cover validation when using `IOptions<T>`

Reply:
Done